tpdatasrc/tpgamefiles/scr/tpModifiers/stunning_fist_extender.py

```diff
@@ -0,0 +1,39 @@
+from templeplus.pymod import PythonModifier
+from toee import *
+import tpdp
+
+print "Registering Stunning Fist Charge Handler"
+
+# Extending Stunning Fist, so new feats can interact with the Stunning Fist Charges
+
+def extraStunningFistCharges(attachee, args, evt_obj):
+    additionalStunningCharges = attachee.d20_query("PQ_Get_Extra_Stunning_Fist_Charges")
+    args.set_arg(0, args.get_arg(0) + additionalStunningCharges)
+    return 0
+
+def deductStunningFistCharge(attachee, args, evt_obj):
+    chargesToDeduct = evt_obj.data1
+    chargesLeft = args.get_arg(0)
+    if chargesLeft:
+        chargesLeft -= chargesToDeduct
+        args.set_arg(0, chargesLeft)
+    return 0
+
+def getStunningFistCharges(attachee, args, evt_obj):
+    chargesLeft = args.get_arg(0)
+    evt_obj.return_val = chargesLeft
+    return 0
+
+def stunningFistTooltip(attachee, args, evt_obj):
+    #Adding a tooltip, so you can see how many charges are left
+    chargesLeft = args.get_arg(0)
+    evt_obj.append("Stunning Fist uses left: {}".format(chargesLeft))
```

Comment:
Can you try using a string from .mes file here? Also less verbosity

Reply:
Actually, please remove this tooltip, it should only appear in the radial menu. I don't want the clutter.

Reply:
I fully agree with you, I always set charges in the radialmenu in the form of (chargesLeft/maxCharges). But the radial is done by the game atm, I would need to replace it, to add the charges to the radial correct?

Reply:
Yeah i thought so.
I think I'll add an option to replace existing hooks.
Alternatively you could rewrite the whole condition from scratch if you're up to it. I think @dolio was working on it?